packages/checkbox/migration.md

```diff
@@ -0,0 +1,96 @@
+# Migration Notes
+
+## Changes
+
+Checkbox
+
+- Support for the `variantColor` prop has been deprecated. Use `colorScheme` prop instead.
+
+For example, `<Checkbox colorScheme="blue">Option</Checkbox>`
+
+```jsx
+// before
+<Checkbox variantColor="blue">Option</Checkbox>
+
+// after
+<Checkbox colorScheme="blue">Option</Checkbox>
+```
+
+- Support for the `isFullWidth` prop has been deprecated. The Checkbox takes up the
+  width of the parent by default.
+
+CheckboxGroup
+
+- To reduce the API surface, we're deprecating the `isInline` prop in favor of `direction` prop for the orientation of the checkbox group
+
+- The `direction` prop takes either `row` or `column` orientation.
+
+```jsx
+// before
+<CheckboxGroup isInline defaultValue={["one", "two"]}>
+  <Checkbox value="one">One</Checkbox>
+  <Checkbox value="two">Two</Checkbox>
+  <Checkbox value="three">Three</Checkbox>
+</CheckboxGroup>
+
+// before
+<CheckboxGroup direction="row" defaultValue={["one", "two"]}>
+  <Checkbox value="one">One</Checkbox>
+  <Checkbox value="two">Two</Checkbox>
+  <Checkbox value="three">Three</Checkbox>
+</CheckboxGroup>
+```
+
+## Features
+
+Checkbox
+
+- Support for `iconColor` prop to customize the color of the check icon
+
+```jsx
+<Checkbox iconColor="blue">Option</Checkbox>
+```
+
+- Support for `iconSize` prop to customize the size of the check icon
+
+```jsx
+<Checkbox iconSize="1rem">Option</Checkbox>
+```
+
+- Support for `iconSize` prop to customize the size of the check icon
+
+```jsx
+<Checkbox labelSpacing="1rem">Option</Checkbox>
+```
+
+- The `useCheckbox` hook is exported with state and focus management logic for use in create tailor made checkbox component for your application
```

Comment:
```suggestion
- The `useCheckbox` hook is exported with state and focus management logic for use in creating tailor-made checkbox components for your application
```

Reply:
Thanks for catching these @with-heart

Reply:
No prob! Feel free to assign me as a reviewer on whatever. I'll gladly look over anything you throw my way :)